src/version/features/ContrastMode/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+
+import IconButton from '@material-ui/core/IconButton';
+import ContrastIcon from '@material-ui/icons/Tonality';
+
+import { contrastModeAction } from "../../actions/contrastModeAction";
+
+/**
+ * Thic component returns Contrast Mode button
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ */
+class ContrastMode extends Component {
+
+    state = {
+        isContrastMode: this.props.contrastMode,
```

Comment:
is it in the global state or in the local state? There should be only one source of truth.

Reply:
Changed to: 
state = {
        isContrastMode: false,
}